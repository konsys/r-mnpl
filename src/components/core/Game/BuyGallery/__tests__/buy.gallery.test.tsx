import BuyGallery from "../BuyGallery";
import React from "react";
import { shallow } from "enzyme";
describe("Buy gallery test", () => {
  it("should render", () => {
    expect(shallow(<BuyGallery />)).toMatchSnapshot();
  });
});
