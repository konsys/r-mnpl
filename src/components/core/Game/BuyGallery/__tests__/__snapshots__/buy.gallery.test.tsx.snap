// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Buy gallery test should render 1`] = `ShallowWrapper {}`;
